ui/static/ui/js/utils.jsx

```diff
@@ -264,6 +367,19 @@ define("utils", ["jquery", "lodash", "react", "react_infinite", "select2"],
     StatusBox: StatusBox,
     ICheckbox: ICheckbox,
     Select2: Select2,
-    InfiniteList: InfiniteList
+    InfiniteList: InfiniteList,
+    showConfirmationDialog: function(options, containter) {
```

Comment:
Actually, can you change this to return just `ConfirmationDialog` instead? The caller can call `React.render` themselves